package.json

```diff
@@ -142,8 +143,11 @@
     "react": "*",
     "react-intl": "^5.7.0 ",
     "react-query": "^3.6.0",
+    "react-redux": "^7.2.0",
     "react-router": "^5.2.0",
-    "react-router-dom": "^5.2.0"
+    "react-router-dom": "^5.2.0",
+    "redux": "^4.0.0",
+    "redux-form": "^8.3.0"
```

Comment:
Do you _need_ to use redux-form here? AFAI can tell, [EditableList lets you use final-form](https://github.com/folio-org/stripes-smart-components/blob/master/lib/EditableList/EditableListFinalForm.js).

Reply:
I couldn't get it to work in final-form variant.